### PR TITLE
Add features section

### PR DESCRIPTION
**Pull request by emillon**
- Dune knows OCaml and its ecosystem
- Dune is fast
- Dune is widely used
- add an example

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -5,43 +5,43 @@ layout: base
...
+Using it, you can build executables, libraries, run tests, and much more.
+
 <div class="flex-grid">
 <div class="col text-container">
-#### Great thing
-Dune does amazing things that will change your life. You will
-wonder what you did without it and thank the OCaml wizard every
-day.
+#### Dune knows OCaml and its ecosystem
+Dune has rules that precisely capture how the different compilers work.
```
Comment (diml) [inline index.md]: When I read this sentence it made me think that Dune supports several different languages. What compilers are you thinking of?

File: index.md [hunk 2]
```diff
@@ -5,43 +5,43 @@ layout: base
...
+Dune works hard to do things once and in parallel, so both cold and incremental
+builds are way faster than traditional build systems.
 
-#### Most wonderful thing
-Dune does amazing things that will change your life. You will
-wonder what you did without it and thank the OCaml wizard every
-day.
+#### Dune is widely used
+Dune is used in both large projects and small libraries.
+About 30% of OPAM packages are built using Dune.
```
Comment (diml) [inline index.md]: Maybe we should give examples of big projects using dune, such as lwt or coq

File: index.md (hunk 1 above)
Comment (emillon) [reply index.md]: `ocamlc` and `ocamlopt`, but yes that's confusing. I'll rework that part.

File: index.md (hunk 2 above)
Comment (emillon) [reply index.md]: coq doesn't seem to be using dune. maybe we can include mirage?

Comment (emillon) [reply index.md]: or just leave that out and we'll be put that in "featured uses" below.

Comment (diml) [reply index.md]: There is a PR to switch coq to dune, I believe coq developers are waiting for the 1.0.0, but yh we should wait until it's merged. Putting it in featured uses seems good

**Comment by emillon** [comment]
Thanks!

**Pull request merged**